sentry-errors.md

```diff
@@ -0,0 +1,19 @@
+Process for triaging new errors on Sentry
+=========================================
+
+Make it part of your process to go to Sentry and go to the "Needs Triage" view of the production project.
+These are all the issues that nobody looked at yet. Check them out and do the following.
+
+* It is an error that can be fixed:
+  * Create a GH issue and link the Sentry error to it.
```

Comment:
we can only link one repo in the sentry interface as far as i can tell, but of course you can just create an issue in mantecao and link back to sentry.

I think we should do it as you say and create an issue in the resmio repo for updating mantecao.  As soon as that is closed we can set the sentry error to 'fixed in next release'.